playbook-website/app/javascript/site_styles/_visual_guidelines.scss

```diff
@@ -42,4 +45,68 @@
     .nine { margin-left: 400px; }
     .ten { margin-left: 450px; }
   }
-}
\ No newline at end of file
+
+
+#border_radius_container{
+  margin: 0px 10px 0px 0px !important;
```

Comment:
Let's use our spacing variables or we can use the margin classes on the actual div themselves and delete it from here.